pilot/pkg/proxy/envoy/v1/watcher.go

```diff
@@ -81,6 +81,10 @@ const (
 	azRetryAttempts = 10
 )
 
+// pilotListenerAddress is the proxy listener to pilot
+// It allows code running in the istio-proxy container (UID) to make a request to pilot even with mtls enabled
+var pilotListenerAddress = "127.0.0.1:14999"
```

Comment:
I asked in other PRs: when adding new ports, please stay in the 15000 range, and allocate them in sequence. It is very convenient to have a range of ports instead of random.

Reply:
@costinm Fair enough. Which port would you suggest?

Reply:
14999 isn't random